### ISSUE TITLE
Feature : want to add functionality of Forget password. 

### ISSUE THREAD

**Issue by VinayLodhi1712** [Luson045/medi-connect#283]
I want to add functionality  of Forget password. So if user forget the password we give him an option to forget the password and set the new password and the new password should store on database. For this i need to design frontend and backend part both in which frontend part i can do easily and even backend can also done easily but you have to help me in setup of database of  mongodb and help in .env.example file setup. 

![forget password](https://github.com/user-attachments/assets/14c4694d-971b-40f8-b7d7-82ce2a160b60)
![forgotpassword](https://github.com/user-attachments/assets/76784742-4f7c-4de6-86eb-58852e8e5e82)

assign this issue under gssoc-extended and hactoberfest

**Comment by geetanjalidubey9** [comment]
pls assign this task to me

**Comment by Luson045** [comment]
I'll assign this issue in a few days, as currently i cannot get engage with development due to my exams.. @VinayLodhi1712

**Comment by Luson045** [comment]
@VinayLodhi1712 yeah that's how it will work, you can make a temporary token that is sent to the email and that token is used for verification. For sending email there is already the back-end code, you need to make an api route to handle verification logic. And send request to that from backend, also make sure to make the api in the auth folders.. that's it.
